cli_tools/gce_vm_image_import/importer/bootable_processor.go

```diff
@@ -0,0 +1,131 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"context"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+
+	daisy_utils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/daisy"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+)
+
+type bootableProcessor struct {
```

Comment:
you may have discussed this topic since I didn't read old commits - would "bootable_disk_process" be a bteer name since another one is named "data_disk_processor"?

Reply:
sounds good to me! Changed bootableProcessor to bootableDiskProcessor (type names and file names)